### PR TITLE
[WFCORE-1628] add --absolute-resources parameter to module add CLI co…

### PR DESCRIPTION
**Pull request by simkam**
…mmand + doc + tests

https://issues.jboss.org/browse/WFCORE-1628

### CODE REVIEW

**Comment by bstansberry** [comment]
@aloubyansky or @jfdenise please approve when you get a chance.

File: cli/src/main/resources/help/module.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ SYNOPSIS
   To add a module locally
 
     module add [--module-root-dir=modules_root_directory_path] --name=module_name 
-               [--slot=slot_name] --resources=list_of_jars [--resource-delimiter=delimiter]
+               [--slot=slot_name] --(absolute-)resources=list_of_jars [--resource-delimiter=delimiter]
```
Comment (aloubyansky) [inline cli/src/main/resources/help/module.txt]: Could you please list the new argument separately next to the resources argument?
It's quite tricky to express the dependencies between the arguments here in this syntax. I'm wondering whether it'd be less confusing to mark both of them as optional.
Thanks for the contribution!

**Comment by simkam** [comment]
updated help text, second commit should fix test failures, I'll squash if tests pass
@aloubyansky is change in second commit ok?

File: cli/src/main/resources/help/module.txt [hunk 2]
```diff
@@ -3,7 +3,7 @@ SYNOPSIS
   To add a module locally
 
     module add [--module-root-dir=modules_root_directory_path] --name=module_name 
-               [--slot=slot_name] --(absolute-)resources=list_of_jars [--resource-delimiter=delimiter]
+               [--slot=slot_name] [--resources=list_of_jars] [--absolute-resources=list_of_jars] [--resource-delimiter=delimiter]
```
Comment (aloubyansky) [inline cli/src/main/resources/help/module.txt]: The content looks good to me. But purely for formatting reasons, please, don't make lines longer than 80 characters. It won't break the CLI but just to be consistent with the other help files. Thanks!

Comment (simkam) [inline cli/src/main/resources/help/module.txt]: ok, added new line, it still exceeds 80 characters a bit, but it the line was longer even before my changes

**Comment by aloubyansky** [comment]
Looks good to me. Thanks!

**Comment by simkam** [comment]
squashed

**Comment by bstansberry** [comment]
I removed the Ready for Merge label because I see there is the WildFly full test PR. That test needs to be made part of this PR.

**Comment by simkam** [comment]
added new test, squashed, rebased

**Pull request merged**